taxonomy-editor-frontend/src/pages/editentry/FetchAndDisplayRelations.jsx

```diff
@@ -0,0 +1,41 @@
+import useFetch from "../../components/useFetch";
+import { Box, Typography } from "@mui/material";
+import { Link } from "react-router-dom";
+
+const FetchRelations = ({url, title}) => {
```

Comment:
this component could use another name, as well as the file. Fetching the info is in this case just an implementation detail